qa/standalone/scrub/osd-scrub-snaps.sh

```diff
@@ -101,11 +101,13 @@ function create_scenario() {
     OBJ5SAVE="$JSON"
     # Starts with a snapmap
     ceph-kvstore-tool bluestore-kv $dir/${osd} list 2> /dev/null > $dir/drk.log
-    grep "^m.*SNA_.*[.]1[.]obj5[.][.]$" $dir/drk.log || return 1
+    grep SNA_ $dir/drk.log
+    grep "^p.*SNA_.*[.]1[.]obj5[.][.]$" $dir/drk.log || return 1
     ceph-objectstore-tool --data-path $dir/${osd} --rmtype nosnapmap "$JSON" remove || return 1
     # Check that snapmap is stil there
     ceph-kvstore-tool bluestore-kv $dir/${osd} list 2> /dev/null > $dir/drk.log
-    grep "^m.*SNA_.*[.]1[.]obj5[.][.]$" $dir/drk.log || return 1
+    grep SNA_ $dir/drk.log
+    grep "^p.*SNA_.*[.]1[.]obj5[.][.]$" $dir/drk.log || return 1
```

Comment:
I would suggest accepting both 'm' & 'p', as to not create a versioning issue

Reply:
Good idea!